tests/model_registry/conftest.py

```diff
@@ -0,0 +1,307 @@
+from ocp_resources.deployment import Deployment
+from ocp_resources.model_registry import ModelRegistry
+from simple_logger.logger import get_logger
+from kubernetes.dynamic import DynamicClient
+
+from tests.model_registry.utils import get_endpoint_from_mr_service, get_mr_service_by_label
+from utilities.infra import create_ns
+from utilities.constants import Protocols
+
+
+LOGGER = get_logger(name=__name__)
+
+DB_RESOURCES_NAME = "model-registry-db"
+
+
+@pytest.fixture(scope="class")
+def model_registry_namespace(admin_client: DynamicClient) -> Generator[Namespace, Any, Any]:
+    # This namespace should exist after Model Registry is enabled, but it can also be deleted
+    # from the cluster and does not get reconciled. Fetch if it exists, create otherwise.
+    ns = Namespace(name="rhoai-model-registries", client=admin_client)
+    if ns.exists:
+        yield ns
+    else:
+        with create_ns(
+            name="rhoai-model-registries",
+            admin_client=admin_client,
+            teardown=False,
+            ensure_exists=True,
+        ) as ns:
+            yield ns
+
+
+@pytest.fixture(scope="class")
+def model_registry_db_service(
+    admin_client: DynamicClient, model_registry_namespace: Namespace
+) -> Generator[Service, Any, Any]:
+    with Service(
+        client=admin_client,
+        name=DB_RESOURCES_NAME,
+        namespace=model_registry_namespace.name,
+        ports=[
+            {
+                "name": "mysql",
+                "nodePort": 0,
+                "port": 3306,
+                "protocol": "TCP",
+                "appProtocol": "tcp",
+                "targetPort": 3306,
+            }
+        ],
+        selector={
+            "name": DB_RESOURCES_NAME,
+        },
+        label={
+            "app.kubernetes.io/name": DB_RESOURCES_NAME,
+            "app.kubernetes.io/instance": DB_RESOURCES_NAME,
+            "app.kubernetes.io/part-of": DB_RESOURCES_NAME,
+            "app.kubernetes.io/managed-by": "kustomize",
+        },
+        annotations={
+            "template.openshift.io/expose-uri": "mysql://{.spec.clusterIP}:{.spec.ports[?(.name==\mysql\)].port}",
+        },
+    ) as mr_db_service:
+        yield mr_db_service
+
+
+@pytest.fixture(scope="class")
+def model_registry_db_pvc(
+    admin_client: DynamicClient,
+    model_registry_namespace: Namespace,
+) -> Generator[PersistentVolumeClaim, Any, Any]:
+    pvc_kwargs = {
+        "accessmodes": "ReadWriteOnce",
+        "name": DB_RESOURCES_NAME,
+        "namespace": model_registry_namespace.name,
+        "client": admin_client,
+        "size": "5Gi",
+        "label": {
+            "app.kubernetes.io/name": DB_RESOURCES_NAME,
+            "app.kubernetes.io/instance": DB_RESOURCES_NAME,
+            "app.kubernetes.io/part-of": DB_RESOURCES_NAME,
+            "app.kubernetes.io/managed-by": "kustomize",
+        },
+    }
+
+    with PersistentVolumeClaim(**pvc_kwargs) as pvc:
+        yield pvc
```

Comment:
Why use kwargs here instead of passing the arguments to `PersistentVolumeClaim` as usual?

Reply:
I think I copied this from another conftest file and it was defined like this, in the end I'd say it's the same